tensorboard/webapp/widgets/data_table/header_cell_component.scss

```diff
@@ -31,15 +31,39 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
 .cell mat-icon {
   height: 12px;
   width: 12px;
-
-  ::ng-deep path {
-    fill: unset;
-  }
 }
 .cell {
   align-items: center;
   display: flex;
 }
+.cell .sorting-icon-container {
+  ::ng-deep path {
+    fill: unset;
+  }
```

Comment:
Did you check dark mode? You might also need fill unset and you can put them together

Reply:
I did check dark mode, and it looks fine without the fill unset. I separated the fill unset to only apply to the sorting container because it caused issues with rendering the delete icon.